.gitignore

```diff
@@ -22,7 +22,9 @@ habitat/results
 /.direnv
 /.envrc
 results/
-
+.vagrant/
```

Comment:
Can we remove changes to the `.gitignore` file from this PR? If we feel these changes are necessary, I'd like to address them separately. Thanks!